spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -331,14 +340,13 @@ def main(args=None):
     if fname_disc:
         # Apply straightening to disc-label
         printv('\nApply straightening to disc labels...', verbose)
-        run_proc('isct_antsApplyTransforms -d 3 -i %s -r %s -t %s -o %s -n %s' %
+        run_proc('sct_apply_transfo -i %s -d %s -w %s -o %s -x %s' %
```

Comment:
what is the reason for replacing the call to the binary by the call to SCT's script?  historically, we were calling the script, and then it was changed to call the binary directly because execution was faster:
https://github.com/neuropoly/spinalcordtoolbox/commit/32a4decd2291be75b9c22b1caf84d130f4ac1f14#diff-f35ac866e7c1dd2ca3d9bfadff4715823e178618892decb6d03bcb9dcdf4d109

Reply:
nevermind-- the reason is because you wanted to use the feature 'label'